planet/sync/__init__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Planet Labs, Inc.
+# Copyright 2022 Planet Labs PBC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from .data import DataAPI
+from .orders import OrdersAPI
+from .subscriptions import SubscriptionsAPI
+from .client import Planet
+
+__all__ = ['DataAPI', 'OrdersAPI', 'Planet', 'SubscriptionsAPI']
```

Comment:
Nit: the whole idea of Planet is that the user no longer has to consider the other APIs... If they want to, they can import them individually but I think it's confusing the import the "shortcut" (Planet) as well as the thing it shortcuts to...

Reply:
Good idea to keep things simple.  I removed the API clients from the __all__ namespace and just have the `Planet` wrapper.